test/executor_test.rb

```diff
@@ -2,444 +2,455 @@
 require_relative 'test_helper'
 require 'mocha/minitest'
 
+require 'sidekiq'
+require 'sidekiq/api'
+require 'sidekiq/testing'
+::Sidekiq::Testing::inline!
+require 'dynflow/executors/sidekiq/core'
+
 module Dynflow
   module ExecutorTest
-    describe "executor" do
-
-      include PlanAssertions
+    [::Dynflow::Executors::Parallel::Core, ::Dynflow::Executors::Sidekiq::Core].each do |executor|
```

Comment:
I've added both executors to test, only line 17-19 (cleanup of the process_word), 23 (setting the executor) and 24 (setting the process role) has been modified. The rest is just whitespace